e2e/transform/babel-jest-async/transformer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {fileURLToPath} from 'url';
+import babelJest from 'babel-jest';
+
+export default {
+  ...babelJest.default.createTransformer({
```

Comment:
not too happy about this one 🙁 We should output ESM from Jest 28 or something

Reply:
I think adding `export { createTransformer }` to babel-jest would be enough.

Reply:
yep that'd work, but "real" ESM should use a single `default` export of an object, so the issue is a "fake" esm module imported directly in native ESM. Edge casey enough that I don't think it's worth changing

Reply:
Oh, and just to be explicit - the `.default` change is not because of _this_ PR, but the change made in #11193